fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -460,7 +484,7 @@ protected Action buildActionChain() {
      * @throws FHIRGeneratorException
      */
     private com.ibm.fhir.persistence.jdbc.dto.Resource createResourceDTO(String logicalId, int newVersionNumber,
-            Instant lastUpdated, Resource updatedResource) throws IOException, FHIRGeneratorException {
+            Instant lastUpdated, Resource updatedResource, String resourcePayloadKey) throws IOException, FHIRGeneratorException {
```

Comment:
add the new param to the method javadoc as well

Reply:
ok